pygmt/tests/test_dimfilter.py

```diff
@@ -0,0 +1,76 @@
+"""
+Tests for dimfilter.
+"""
+import os
+
+import pytest
+import xarray as xr
+from pygmt import dimfilter, load_dataarray
+from pygmt.datasets import load_earth_relief
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import GMTTempFile
+
+
+@pytest.fixture(scope="module", name="grid")
+def fixture_grid():
+    """
+    Load the grid data from the sample earth_relief file.
+    """
+    return load_earth_relief(
+        resolution="01d", registration="pixel", region=[124, 130, -25, -20]
+    )
```

Comment:
Please change the tests to use the `static_earth_relief` data.

Reply:
done in 8cb34e0